blueprints/model/files/__root__/__path__/__name__.js

```diff
@@ -1,5 +1,5 @@
 <%= importStatements %>
 
 export default Model.extend({
-  <%= attrs %>
+<%= attrs.length ? '  ' + attrs : '' %>
```

Comment:
I personally think that using an `if` here, might make this more readable:

``` js
export default Model.extend({
<% if (attrs) { %>
  <%= attrs %>
<% } %>
});
```

What do you think?

---

Another possibility would be to prepend the empty string within `blueprints/model/index.js`. We are already doing that for the [`needs` statement](https://github.com/emberjs/data/blob/afeb5e052b97b3113d993965d205bb725a742a63/blueprints/model/index.js#L82), so I think this would be fine for `attrs` too...